agent_build/tools/environment_deployments/steps/cache_lib.sh

```diff
@@ -1,99 +0,0 @@
-#!/bin/bash
```

Comment:
@Kami This file just moved to `agent_build/tools/environment_deployments/steps/step_runner.sh` I am not really sure why GitHub treats it as deleted and new files instead of renamed.